spec/requests/forms_controller_spec.rb

```diff
@@ -114,7 +114,7 @@
     end
 
     context "with a user with a trial account" do
-      let(:user) { build(:user, role: :trial) }
+      let(:user) { build(:user, :with_trial_role, id: 123) }
```

Comment:
What happens if a user ID is not set here?

Reply:
The factory for the user never sets `id`, so the form won't have a `creator_id` set, which means the "you don't have access to this page" page is shown.

Reply:
Ah, interesting! That makes sense, thanks for explaining. But then, why were the tests were passing before? 🤔

Reply:
So that's interesting. It turns out that the default organisation for the form and user factories is the same. Previously we only specified the role, but now with `with_trial_role` we also specify no organisation. It previously worked because if the user's org and form's org match it shows the form (app/policies/form_policy.rb:33), even though the user and creator IDs are nil 😬

Reply:
Ah interesting. But that's good though! It means your changes have tightened up the tests, and we're now getting a more accurate check from this test. Thanks for checking.

Reply:
It's interesting that a trial user with an org set can see forms from that org... I wonder if we should change that.

Reply:
Yes it has opened up that question. My instinct is that is incorrect...

Reply:
I've created this card to track fixing this: https://trello.com/c/hUoW5S1e

I'll fix in a subsequent PR.